test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -821,6 +826,116 @@ def create_pytorch_only_extra_kwargs():
             create_pytorch_only_extra_kwargs,
         )
 
+    @_beartype.beartype
+    def _test_fake_tensor_mode_exporter(
+        self,
+        model_name: str,
+        create_model: Callable,
+        create_args: Callable,
+        create_pytorch_only_kwargs: Callable,
+    ):
+        """Test helper for large-scale exporter.
+
+        Arguments:
+            model_name: Name of the model. It used to name temporary files.
+            create_model: A function that creates a model. It should always create the same model.
```

Comment:
Does `same model` imply parameter values are also the same?

Reply:
Yes, because `create_model` and `create_args` are called twice in this helper
The first time it is called outside FakeTensorMode and real model and input are saved for the numerical validation.
The second time it is called within `enable_fake_mode` for the actual FX tracing.

In the end, `ExportOutput.save` "merges" the ONNX model without initializers with the initializers found in the first call to `create_model`/`create_args`.

Reply:
But the implementation randomly initializes parameter values, so parameter values are not the same for the models from two invokation.

Not that it matters for the test though.

Reply:
Agreed, it happens and it doesnt matter for us :)
I will remove the "same model" from docstring to kill the ambiguity.